oxidation/libparsec/crates/protocol/src/lib.rs

```diff
@@ -63,11 +63,11 @@ parsec_protocol_cmds_familly!("schema/authenticated_cmds");
 parsec_protocol_cmds_familly!("schema/anonymous_cmds");
 
 pub trait Request<'de> {
-    type Response: Deserialize<'de>;
+    type Response: DeserializeOwned;
```

Comment:
Why do you need `DeserializeOwned` ? If so you can remove the lifetime

Reply:
Sorry for the inconvenience, `Response` should not be rely with the `Request`, I fixed it